### PR TITLE
Changed Komtek name in public pages

### PR DESCRIPTION
**Pull request by Sindre2808**
Change "Kommunikasjonsteknologi" to "Kommunikasjonsteknologi og digital sikkerhet" on public pages. Image on om Abakus page needs an update as well.

### CODE REVIEW

File: app/routes/pages/components/subcomponents/TextWithTitle/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import styles from './TextWithTitle.css';
 
 type Props = {
   title: string,
-  text: string,
+  text: React.component,
```
Comment (SmithPeder) [inline app/routes/pages/components/subcomponents/TextWithTitle/index.js:7]: `import React, {type Node} from 'react'`;

**Comment by Sindre2808** [comment]
image

**Comment by SmithPeder** [comment]
https://github.com/webkom/lego-webapp/blob/21a32f4e767485c43bc791d4ceb750f25d301698/app/routes/app/AppRoute.js#L103

File: app/routes/overview/components/PublicFrontpage.js [hunk 2]
```diff
@@ -182,12 +182,13 @@ class PublicFrontpage extends Component<Props, State> {
...
-                <i className="fa fa-caret-right" /> Kommunikasjonsteknologi
+                <i className="fa fa-caret-right" /> Kommunikasjonsteknologi og
+                digital sikkerhet
               </a>
               <div className={styles.linkDescription}>
                 Vi bruker stadig mer av livene våre på nett, på jobb som i
-                fritid. Kommunikasjonsteknologi brukes etter hvert av alle og
-                overalt.
+                fritid. Kommunikasjonsteknologi og digital sikkerhet brukes
+                etter hvert av alle og overalt.
```
Comment (LudvigHz) [inline app/routes/overview/components/PublicFrontpage.js:191]: Hmm, I feel like we should change this a bit. It doesn't quite sound right :thinking:

**Review by LudvigHz — COMMENTED** [review]
Cool. Shouldn't we change "Datateknologi" to be italic as well?

File: app/routes/overview/components/PublicFrontpage.js (hunk 2 above)
Comment (Sindre2808) [reply app/routes/overview/components/PublicFrontpage.js]: Ok, changed it to "Kommunikasjonsteknologi og digital sikkerhet blir stadig viktigere i en digital verden."

**Review by SmithPeder — APPROVED** [review]

**Pull request merged**